teku/src/test/java/tech/pegasys/teku/cli/options/DepositOptionsTest.java

```diff
@@ -129,23 +131,63 @@ public void shouldSetDefaultBundleSnapshotPathForSupportedNetwork(final String n
                 .toExternalForm());
   }
 
+  @Test
+  public void shouldHaveDepositSnapshotEnabledByDefault() {
+    final String[] args = {};
+    final TekuConfiguration config = getTekuConfigurationFromArguments(args);
+    assertThat(config.powchain().getDepositTreeSnapshotConfiguration().isDepositSnapshotEnabled())
+        .isTrue();
+  }
+
   @Test
   public void shouldIgnoreBundleSnapshotPathForNotSupportedNetwork() {
     final String[] args = {"--network=swift", "--deposit-snapshot-enabled"};
     final TekuConfiguration config = getTekuConfigurationFromArguments(args);
-    assertThat(config.powchain().isDepositSnapshotEnabled()).isTrue();
-    assertThat(config.powchain().getDepositSnapshotPath()).isEmpty();
+    final DepositTreeSnapshotConfiguration depositTreeSnapshotConfiguration =
+        config.powchain().getDepositTreeSnapshotConfiguration();
+    assertThat(depositTreeSnapshotConfiguration.isDepositSnapshotEnabled()).isTrue();
+    assertThat(depositTreeSnapshotConfiguration.getCustomDepositSnapshotPath()).isEmpty();
```

Comment:
bundled?

Reply:
Okay I think I'm getting where I was confused
it's called isDepositSnapshotEnabled, but plays a role of isBundledDepositSnapshotEnabled, but not actually, because in case of custom deposit snapshot it's false (but hey, we still have deposit snapshot enabled). So I think we need either better name or keep too booleans. It's a bit confusing in current way